### DISCUSSION TITLE
Permanently disable disk browser caches (not cleaning)

### DISCUSSION THREAD

**Discussion by eadmaster** [undergroundwires/privacy.sexy#269] [Ideas]
This may improve both privacy and performances on devices with slow disk access.

- in Chrome/Edge -> make sure they are always started with these cmdline params ` --disk-cache-dir=nul --disk-cache-size=0 --media-cache-size=0`
- Firefox -> via `browser.cache.disk.enable` setting
- Internet Explorer (not sure if possible)

**Comment by ItsIgnacioPortal** [comment]
> This may improve both privacy and performances on devices with slow disk access.

You would need storage access times greater than 1000ms for cache-less to be beneficial.

It would make sense to use it for privacy reasons though, if you were trying to protect yourself in the scenario someone got physical access to your device. But in that case, wouldn't it be better to ditch Windows and use Tails instead?
